js/ui/drop_down_button.js

```diff
@@ -685,6 +695,7 @@ let DropDownButton = Widget.inherit({
             case "width":
             case "height":
                 this._buttonGroup.option(name, value);
+                this._updatePopupWidth();
```

Comment:
Let's use `this._popup && this._popup.repaint` instead of it.
And there is no need to update the Popup width when height was changed